litrpc/lit-accounts.proto

```diff
@@ -148,6 +160,22 @@ message UpdateAccountRequest {
     string label = 4;
 }
 
+message UpdateAccountBalanceRequest {
+    // The ID of the account to update. Either the ID or the label must be set.
+    string id = 1;
+
+    /*
+    The label of the account to update. If an account has no label, then the ID
+    must be used instead.
+    */
+    string label = 2;
```

Comment:
should we have an AccountIdentifier type rather since we seem to need this for each account request?

Reply:
Good idea, updated :)